src/common/prompts.go

```diff
@@ -0,0 +1,171 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/manifoldco/promptui"
+	"github.com/opslevel/opslevel-go"
+)
+
+func PromptForCategories(client *opslevel.Client) (*opslevel.Category, error) {
+	list, err := client.ListCategories()
+	if err != nil {
+		return nil, err
+	}
+
+	templates := &promptui.SelectTemplates{
+		Label:    "{{ . }}?",
+		Active:   fmt.Sprintf("%s {{ .Name | cyan }}", promptui.IconSelect),
+		Inactive: "    {{ .Name | cyan }}",
+		Selected: fmt.Sprintf("%s {{ .Name | faint }}", promptui.IconGood),
+	}
+
+	prompt := promptui.Select{
+		Label:     "Select Category",
+		Items:     list,
+		Templates: templates,
+		Size:      len(list),
+	}
+
+	index, _, err := prompt.Run()
+	if err != nil {
+		return nil, err
+	}
+	return &list[index], nil
+}
+
+func PromptForLevels(client *opslevel.Client) (*opslevel.Level, error) {
+	list, err := client.ListLevels()
+	if err != nil {
+		return nil, err
+	}
+
+	templates := &promptui.SelectTemplates{
+		Label:    "{{ . }}?",
+		Active:   fmt.Sprintf("%s {{ .Name | cyan }} ({{ .Index | red }})", promptui.IconSelect),
+		Inactive: "    {{ .Name | cyan }}",
+		Selected: fmt.Sprintf("%s {{ .Name | faint }} ({{ .Index | red }})", promptui.IconGood),
+		Details: `
+		{{ "Alias:" | faint }}	{{ .Alias }}
+		{{ "Description:" | faint }}	{{ .Description }}`,
+	}
+
+	prompt := promptui.Select{
+		Label:     "Select Level",
+		Items:     list,
+		Templates: templates,
+		Size:      len(list),
+	}
+
+	index, _, err := prompt.Run()
+	if err != nil {
+		return nil, err
+	}
+	return &list[index], nil
+}
+
+func PromptForFilter(client *opslevel.Client) (*opslevel.Filter, error) {
+	list, err := client.ListFilters()
+	if err != nil {
+		return nil, err
+	}
+
+	templates := &promptui.SelectTemplates{
+		Label:    "{{ . }}?",
+		Active:   fmt.Sprintf("%s {{ .Name | cyan }}", promptui.IconSelect),
+		Inactive: "    {{ .Name | cyan }}",
+		Selected: fmt.Sprintf("%s {{ .Name | faint }}", promptui.IconGood),
+	}
+
+	noneValue := opslevel.Filter{
+		Name: "None",
+		Id: nil,
+	}
+	list = append([]opslevel.Filter{noneValue}, list...)
+
+	prompt := promptui.Select{
+		Label:     "Select Filter",
+		Items:     list,
+		Templates: templates,
+		Size:      len(list),
+	}
+
+	index, _, err := prompt.Run()
+	if err != nil {
+		return nil, err
+	}
+	return &list[index], nil
+}
+
+func PromptForTeam(client *opslevel.Client) (*opslevel.Team, error) {
+	list, err := client.ListTeams()
+	if err != nil {
+		return nil, err
+	}
+
+	templates := &promptui.SelectTemplates{
+		Label:    "{{ . }}?",
+		Active:   fmt.Sprintf("%s {{ .Name | cyan }}", promptui.IconSelect),
+		Inactive: "    {{ .Name | cyan }}",
+		Selected: fmt.Sprintf("%s {{ .Name | faint }}", promptui.IconGood),
+		Details: `
+		{{ "Aliases:" | faint }}	{{ .Aliases }}
+		{{ "Manager:" | faint }}	{{ .Manager.Name }}`,
+	}
+
+	noneValue := opslevel.Team{
```

Comment:
Pulled this from a "how to prepend to list" that I found but open to feedback if there's a better way to add an object to the front of a list.